src/index.jsx

```diff
@@ -22,14 +22,19 @@ const main = async () => {
   core.endGroup()
 
 
-  const rootPath = core.getInput("root_path") || "./";
+  const rootPath = core.getInput("root_path") || ""; // Micro and minimatch do not support paths starting with ./
```

Comment:
we might want to also remove any leading `./` params, to prevent from breaking existing workflows

Reply:
I thought about this - what do you think about a choice between

1. Quitting the script early if the path starts with `./` and asking the user to remove it
2. Quietly removing it if it's detected

I think doing 2 automatically is a smoother migration because it introduces a quiet side effect, but 1 is simpler for us to maintain. Otherwise, people may get confused when seeing some configs start with `./` and some without.